Notifier-service/.env

```diff
@@ -3,4 +3,5 @@ MONGO_DB_PORT=27017
 MONGO_DB_NAME=Conference_Notify
 SERVER_PORT=3000
 LOG_FOLDER=logs
-LOG_LEVEL=debug
\ No newline at end of file
+LOG_LEVEL=debug
+ORIGIN=["http://localhost:3000", "http://localhost:3020"]
```

Comment:
* Specify only one external host. Arrays wont work here

Reply:
should i remove 3020 then?

Reply:
* I have done the change it was minor.